src/ansys/fluent/core/post_objects/singleton_meta.py

```diff
@@ -0,0 +1,20 @@
+"""Provides a module for Metaclasses."""
```

Comment:
```suggestion
"""Provides a module for metaclasses."""
```
Google search shows it as one word, not capitalized.